subworkflows/nf-core/fasta_build_add_kraken2/meta.yml

```
@@ -0,0 +1,61 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/subworkflows/yaml-schema.json
+name: "fasta_build_add_kraken2"
+description: KRAKEN2 build custom database workflow
```

not sure but just in case I suggest
```suggestion
description: KRAKEN2 build custom database subworkflow
```